core/common/src/test/java/alluxio/metrics/MetricsSystemTest.java

```diff
@@ -54,4 +57,61 @@ public void metricsSystem() {
     sCounter.inc();
     MetricsSystem.stopSinks();
   }
+
+  /**
+   * Tests the "/" "." and "%" special characters can be escaped.
+   */
+  @Test
+  public void testEscape() {
+    AlluxioURI localUri1 = new AlluxioURI("/foo/alluxio/underFSStorage");
+    String localUriEscaped1 = MetricsSystem.escape(localUri1);
+    assertEquals("%2Ffoo%2Falluxio%2FunderFSStorage",
+            localUriEscaped1);
+    assertFalse(localUriEscaped1.contains("/"));
```

Comment:
This is redundant - the `assertEquals` already checked that `localUriEscaped1` does not have `/`.

Same goes to the other `assertFalse` checks

Reply:
Redundant assertFalse() removed